views/offramp/offramp-name.pug

```diff
@@ -0,0 +1,20 @@
+extends ../base
+include ../_includes/banner
+
+block variables
+  -var title = __('Continue with an incorrect name on file')
```

Comment:
This title suggests that you can continue using this service somehow. Is there no better way to word this? @brianhendrick1.